composer.json

```diff
@@ -1,9 +1,13 @@
 {
-    "name": "albertcht/invisible-recaptcha",
+    "name": "danmatthews/invisible-recaptcha",
```

Comment:
This shouldn't be included in the PR.

Reply:
Hi @Jamesking56 - yep i created it to temporarily publish for use in a production app, will remove.